tests/test_config_edge_cases.py

```
@@ -314,6 +323,239 @@ def test_load_config_no_files_found(self):
+                self.assertEqual(saved_creds["access_token"], "secret_token")
+            finally:
+                config_module.relay_config = original_relay_config
+                config_module.config_path = original_config_path
+
+    def test_save_credentials_exception_handling(self):
+        """Test save_credentials handles exceptions gracefully."""
+        credentials = {"user_id": "test"}
+
+        with patch("mmrelay.config.os.makedirs", side_effect=OSError("Disk full")):
+            with patch("mmrelay.config.logger") as mock_logger:
+                result = save_credentials(credentials)
```

Ruff (0.14.14)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)